src/utils/buildExtractStylesLoader.js

```diff
@@ -35,5 +35,5 @@ Error: ${error}
       .map(loader => `${loader}!`)
       .join('')
   );
-  return ExtractTextPlugin.extract({ fallbackLoader, loader: restLoaders });
+  return ExtractTextPlugin.extract({ fallback: fallbackLoader, use: restLoaders });
```

Comment:
We have [this ongoing PR](https://github.com/shakacode/bootstrap-loader/pull/249). TBH I haven't looked into details of the issue and not sure why returned value must be a string (when it's changed to array in ETP). Can you check if this resolves https://github.com/shakacode/bootstrap-loader/issues/238 or not.